pkg/state/aliases.go

```diff
@@ -258,3 +258,35 @@ func (a *aliases) disabledAliases() (map[string]struct{}, error) {
 	}
 	return als, nil
 }
+
+func (a *aliases) aliasesByAddr(addr *proto.WavesAddress) ([]string, error) {
```

Comment:
Use just `proto.WavesAddress`, not `*proto.WavesAddress`. Pointer can be `nil`, struct can't.